metacluster/include/metacluster/MetaclusterConsistency.actor.h

```diff
@@ -259,7 +260,7 @@ class MetaclusterConsistencyCheck {
 					ASSERT_EQ(metaclusterEntry.tenantState, TenantState::READY);
 					ASSERT(entry.tenantName == metaclusterEntry.tenantName);
 				} else if (entry.tenantName != metaclusterEntry.tenantName) {
-					ASSERT(entry.tenantName == metaclusterEntry.renameDestination);
+					ASSERT(entry.tenantName == metaclusterEntry.renameDestination.get());
```

Comment:
We should also verify that the rename destination is present:

```suggestion
					ASSERT(metaclusterEntry.renameDestination.present() && entry.tenantName == metaclusterEntry.renameDestination.get());
```